src/test/java/pixelitor/layers/TextLayerTest.java

```diff
@@ -177,4 +172,19 @@ public void commitSettings() {
 
         iconUpdates.check(0, 0);
     }
+
+    /**
+     * Newly added testCase
+     */
+    @Test
+    public void testHasRasterThumbnail() {
+        TextLayer textLayer = new TextLayer(comp);
+        boolean hasRasterThumbnail = textLayer.hasRasterThumbnail();
```

Comment:
This seems like a useless testcase, because hasRasterThumbnail() is not an important method, it's more like an implementation detail.